consumergroup/consumer_group.go

```diff
@@ -280,9 +295,13 @@ func (cg *ConsumerGroup) topicListConsumer(topics []string) {
 
 		for _, topic := range topics {
 			cg.wg.Add(1)
+			cg.nGoroutines += 1
 			go cg.topicConsumer(ctx, cancel, topic, cg.messages, cg.errors)
 		}
+		//note let's just wait here for test instead:
+		cg.wg.Wait()
```

Comment:
I would suggest us to remove this wait here as it prevents us to receive messages on the `consumerChanges` channel.